site/app/controllers/grading/SimpleGraderController.php

```diff
@@ -281,7 +281,10 @@ public function save($gradeable_id) {
 
         $this->core->getQueries()->saveTaGradedGradeable($ta_graded_gradeable);
 
-        return JsonResponse::getSuccessResponse($return_data);
+        return JsonResponse::getSuccessResponse([
+            'data' => $return_data,
+            'date' => $this->core->getDateTimeNow()->format('c')
+        ]);
```

Comment:
Why is the `data` field being used here?  `getSuccessResponse()` already adds a top-level data field, so it seems odd to structure the response with two nested `data` fields.